tools/project_manager/marc_inspector.php

```diff
@@ -0,0 +1,129 @@
+<?php
+$relPath = '../../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'theme.inc');
+
+require_login();
+
+$rec = @$_REQUEST['rec'];
+if (!$rec) {
+    throw new UnexpectedValueException("Unexpected rec `$rec`");
+}
+
+$record = unserialize(gzdecode(base64url_decode($rec)));
+
+$title = _("MARC record inspector");
+output_header($title, NO_STATSBAR);
+
+echo "<h1>$title</h1>\n";
+echo "<a href='https://www.loc.gov/marc/bibliographic/'>MARC 21 Format for Bibliographic Data</a>\n";
+echo "<p>\n";
+
+$loc_bib = "https://www.loc.gov/marc/bibliographic";
+$field_info = [
+    "leader" => ["tooltip" => "Record header",  "url" => "$loc_bib/bdleader.html"],
```

Comment:
I deliberately didn't translate any of this because, with the exception of the leader (which, TBH, I should probably change back) they're taken verbatim from the titles in the MARC 21 spec.

Reply:
Makes sense, let's add a comment to that effect please.

Reply:
Done.

Reply:
Agreed. Not everything needs to be translated,